scripts/examples/index.csv

```diff
@@ -0,0 +1,34 @@
+# "Relative Path Regex", "Board Type Regex", "Sensor Type Regex", "Flatten Regex"
+"HelloWorld/blinky.py", ".+", ".+", ""
+"HelloWorld/helloworld.py", ".+", "^(?!None).*$", ""
+"Camera/00-Snapshot", ".+", "^(?!None).*$", ""
+"Camera/01-Video-Recording", ".+", "^(?!None).*$", ""
+"Camera/02-Optical-Flow", ".+", "^(?!None).*$", ""
+"Camera/03-Event-Cameras", ".+", "(FROGEYE2020)", ""
+"Camera/04-Global-Shutter", ".+", "(MT9V0X2|MT9V0X2-C|MT9V0X4|MT9V0X4-C)", ""
+"Camera/05-FLIR-Lepton", ".+", "(LEPTON|LEPTON-1.5|LEPTON-1.6|LEPTON-2.0|LEPTON-2.5|LEPTON-3.0|LEPTON-3.5)", ""
+"Camera/06-Time-of-Flight", ".+", "^(?!None).*$", ""
+"Camera/07-Sensor-Control", ".+", "^(?!None).*$", ""
+"Camera/08-Readout-Control", ".+", "^(?!None).*$", ""
+"Image-Processing", ".+", "^(?!None).*$", ""
+"Machine-Learning/00-TensorFlow", ".+", "^(?!None).*$", ""
+"Machine-Learning/01-ST-CubeAI", "(OPENMV2|OPENMV3|OPENMV4|OPENMV4P|OPENMVPT|ARDUINO_GIGA|ARDUINO_PORTENTA_H7|ARDUINO_NICLA_VISION)", "^(?!None).*$", ""
+"Machine-Learning/02-Haar-Cascade", ".+", "^(?!None).*$", ""
+"Barcodes", ".+", "^(?!None).*$", ""
+"Feature-Detection", ".+", "^(?!None).*$", ""
+"April-Tags", ".+", "^(?!None).*$", ""
+# STM32 libraries
+"Interface-Library/00-Arduino", "(OPENMV2|OPENMV3|OPENMV4|OPENMV4P|OPENMVPT|ARDUINO_GIGA|ARDUINO_PORTENTA_H7|ARDUINO_NICLA_VISION)", ".+", ""
+"Interface-Library/01-Pixy-Emulation", "(OPENMV2|OPENMV3|OPENMV4|OPENMV4P|OPENMVPT|ARDUINO_GIGA|ARDUINO_PORTENTA_H7|ARDUINO_NICLA_VISION)", ".+", ""
+"Interface-Library/02-MAVLink", ".+", ".+", ""
+"Interface-Library/03-Modbus", ".+", ".+", ""
+"RPC-Library", ".+", ".+", ""
+"WiFi/(?!WINC1500).*$", "^(?!arduino_nano_33_ble_sense).*$", ".+", ""
+"WiFi/WINC1500", "(OPENMV2|OPENMV3|OPENMV4|OPENMV4P|OPENMVPT)", ".+", ""
+"Bluetooth", "(OPENMV_RT1060|ARDUINO_GIGA|ARDUINO_PORTENTA_H7|ARDUINO_NICLA_VISION|ARDUINO_NANO_RP2040_CONNECT)", ".+", ""
```

Comment:
Make this line:

"examples/10-Bluetooth", "(OPENMV_RT1060|ARDUINO_GIGA|ARDUINO_PORTENTA_H7|ARDUINO_NICLA_VISION)", ".+", ""